test/EVMHost.cpp

```diff
@@ -24,6 +24,10 @@
 #include <test/evmc/helpers.hpp>
 #include <test/evmc/loader.h>
 
+#include <test/Options.h>
```

Comment:
Is this actually used anywhere in this file? It pulls in the boost unit test headers (which might do some static magic) - and this might be connected to the ossfuzz linking errors.

Reply:
My IDE warns me that this header is unused. After removing it, I was able to build locally. Hence, updated this PR.